README.md

```diff
@@ -43,4 +43,8 @@ This installer is incompatible with `default` namespace installations, install a
 
 ## GitOps/ArgoCD Configuration
 
-Detailed documentation for configuring GitOps/ArgoCD can be found in [`GITOPS-CONFIG.md`](./docs/GITOPS-CONFIG.md)
\ No newline at end of file
+Detailed documentation for configuring GitOps/ArgoCD can be found in [`GITOPS-CONFIG.md`](./docs/GITOPS-CONFIG.md)
```

Comment:
Should we provide a bit more info on setting up ENVs? Glancing at the README, I could not find any mention of how to setting up https://github.com/michael-valdron/ai-rhdh-installer/blob/feat/pipelines-config/default-private.env 🤔

Reply:
If this is TODO, please feel free to merge PR with other LGTMs received

Reply:
Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/21c11e362b2e1fd47ee59025c77d1360c7cee3e7